docs/Technical-Documentation/Architecture-Decision-Record/016-security-scan-waiver.md

```diff
@@ -22,6 +22,26 @@ When investigating, we sought out information on what other, modern ACF systems
 
 While we also investigated and developed technical solutions utilizing either React-app-rewired or Next.js, we decided that the departure from our existing stack was not a worthwhile investment and could lead to other complications down the road.
 
+### 2024 Update
```

Comment:
@jtimpe @raftmsohani are any of the CSP related findings on the latest nightly scan related to what is described below? if so, we may want acknowledge that this is not just being found in webinspect but zap as well. 

https://app.circleci.com/pipelines/github/raft-tech/TANF-app/24238/workflows/3faab2a5-dab3-4f52-bf56-6e321dfb9b03/jobs/71676/artifacts

Reply:
i do not believe any of those findings are related to this waiver

Reply:
@ADPennington Jan is correct, they are not relevant.

On a separate note, I am creating a ticket to check for user auth, similar to what we do for Kibana auth, to prevent access to api without auth: https://tdp-frontend-develop.acf.hhs.gov/v1/users/id/